manifests/charts/cluster-operator/templates/webhooks.yaml

```diff
@@ -28,3 +28,28 @@ webhooks:
     resources:
     - clusters
   sideEffects: None
+- admissionReviewVersions:
+  - v1
+  - v1beta1
```

Comment:
nit:   we can not support v1beta1, v1 is there since very early, need to check which k8s version

Reply:
I have confirmed that as of Kubernetes v1.22.0, the v1beta1 version of the Admissionregistration.k8s.io API is indeed deprecated and removed, leaving only the v1 version in use.

 This change is detailed in the [Kubernetes 1.22: API Changes blog post ](https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/)on the official Kubernetes website. 

Therefore, it would be appropriate for our project to only support the v1 version.

Reply:
Considering that other third-party libraries we are using have both support v1beta1and v1, such as [kubeadmcontrolplanes](https://github.com/Xieql/kurator/blob/58e1e44c2b01f93ee442e51ccefc89a8264f7d6f/manifests/charts/cluster-operator/templates/capi-kubeadm-control-plane-validating-webhook-configuration.yaml#L12), 

perhaps we could keeping it as is to maintain consistency with other third-party libraries we are using